firebase-perf/src/test/java/com/google/firebase/perf/FirebasePerformanceTest.java

```diff
@@ -573,6 +574,7 @@ private FirebasePerformance initializeFirebasePerformancePreferences(
         mock(FirebaseInstallationsApi.class),
         transportFactoryProvider,
         spyRemoteConfigManager,
+        mock(FirebaseSessions.class),
```

Comment:
How well does this mock? I thought classes in Kotlin were `final` by default and couldn't be mocked?

Reply:
Firebase Performance depends on [Mockito-Inline](https://github.com/firebase/firebase-android-sdk/blob/master/firebase-perf/firebase-perf.gradle#L124) which allows for Kotlin final classes to be mocked. [Reference](https://stackoverflow.com/a/63233687)